### DISCUSSION TITLE
Proposal:  Add "maximumRemotePercentage" attribute to <podcast:value>

### DISCUSSION THREAD

**Discussion by agates** [Podcastindex-org/podcast-namespace#607] [Enhancement Proposal]
This proposal is to add a new attribute, `maximumRemotePercentage`, to the `<podcast:value>` tag.

* maximumRemotePercentage (optional) - The maximum desired `remotePercentage` requested if the value block is referenced in `<podcast:valueTimeSplit>` or `<podcast:valuePercentage>` in another feed. If not defined, defaults to 100. If the value is less than 0, 0 is assumed.
If the value is greater than 100, 100 is assumed.

**Comment by thebells1111** [comment]
I'm commenting to bump this. As more people are starting to use 'publisher', 'podcastL', and 'musicL' feeds, there should be a mechanism in place for podcasters/musicians to say "I'm willing to share a percentage of my value, but no more than this percent", or if they don't want to share any of their value, they shouldn't be forced to because a curator simply added their song to a playlist.

**Reply by samsethi** [reply]
I get the issue from the creator's perspective.  But apps like TrueFans or Fountain etc. take 3% of all transactions to cover our costs and make a small revenue.  If the creator says they only want to share 1% maximum or not share their value, should we remove/hide them from our apps?

**Reply by jamescridland** [reply]
> "I'm willing to share a percentage of my value, but no more than this percent"

I'm with Sam on this. It's not "your" value if you are part of a playlist - the curator of the playlist is also part of the value chain.

Where it gets a little hard is - what happens if a curator or app wants to take 30%? Do we hope that listeners are savvy about this and switch away from the app?

**Reply by thebells1111** [reply]
For arguments sake, because Podcast Weekly Review is in feed, should I be allowed to create a 'News Makers' playlist that includes an episode of PWR I really like, then take 5% of the value whenever someone plays your show from my feed? What about 50%? What about 95%?

Should you as the podcast host get to decide if someone adding one of your episodes to their playlist gets to add themselves to your value stream for the simple task of curating your episode? If you do think they should be able to inject themselves into your value stream, do you think you should be able to say what percentage they're allowed?

**Reply by thebells1111** [reply]
> If the creator says they only want to share 1% maximum or not share their value, should we remove/hide them from our apps?

This depends on how you're handling your 3%. If a listener boosts 100 sats, are you taking 3 sats from the boost and sending 97 to the podcaster, or are you taking 103 sats from the listener and sending 100 sats to the Podcaster? Who's responsible for supporting True Fans, the podcasters or the listeners? I would argue the listeners, and you can charge your 3% to them regardless of what the podcaster wants, because you're not taking any of the boost amount off the top of the amount being sent to the podcaster.

**Reply by jamescridland** [reply]
@thebells1111 Your answer to https://github.com/Podcastindex-org/podcast-namespace/discussions/607#discussioncomment-11105061 is probably made in https://github.com/Podcastindex-org/podcast-namespace/discussions/607#discussioncomment-11105103

If the additional value is marked as a "fee" - so if you support PWR by boosting 1000 sats, you're actually charged 1030 with the 30 clearly going to the playlist curator - then I think that's fine.

If the additional value is a share of the value, I can see your point.

This all comes back to transparency, I think.

**Reply by agates** [reply]
> I get the issue from the creator's perspective. But apps like TrueFans or Fountain etc. take 3% of all transactions to cover our costs and make a small revenue. If the creator says they only want to share 1% maximum or not share their value, should we remove/hide them from our apps?

This specification is entirely unrelated to app fees, which are not specified in feeds in the first place.  We can clarify that if needed.

**Comment by samsethi** [comment]
At the moment TrueFans gives 1% of the SATs paid IF someone plays a curated playlist.  So if I create my own playlist called News Makers, then the maximum you can earn is 1%.  the remaining 99% goes to the podcast being played.  I think 1% is the reward for the person curating the playlist and then encouraging people to play their playlist. Podcasters should not begrudge this 1% promotion fee.  The playlist curator has picked your podcast and promoting on your behalf to their friends.  

I agree we need to cap what the playlist curator is paid.  We have made this a system wide fixed fee of 1% and have not given the podcaster or curator the option to set their own value.  

I get the aim of this tag but it is probably super early. 

<img width="1365" alt="TrueFans___Dashboard" src="https://github.com/user-attachments/assets/4f0c1b66-342d-40e1-bcf7-8ce499379f05">

**Reply by agates** [reply]
I mostly created this proposal in response to the v4v licensing drama.  Legal documents don't mean anything without a technical way to tell the podcast apps, or other downstream consumers.

Where all that went, I have no idea.  I don't currently have a horse in the race.
